frontend/src/app/App.scss

```diff
@@ -44,3 +49,7 @@ body,
     height: auto;
   }
 }
+
+.pf-v5-c-drawer__body {
```

Comment:
@jenny-s51 Can we note a bug here with PF... they broke this functionality and we should label certain CSS as such so we know we didn't do this ourselves for no reason.

@christianvogt was there an issue on PF that we can reference?

Reply:
Yes @andrewballantyne good call, added a reference to the PF bug opened by @christianvogt yesterday: https://github.com/patternfly/patternfly-react/issues/9769